pizzeria-backend/Models/Interfaces/AddOn.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace pizzeria_backend.Models.Interfaces
+{
+    public interface IAddOn
```

Comment:
nitpick: File name should match class/interface name

Reply:
Done